<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifierprogrammes</class>
 <widget class="QDialog" name="modifierprogrammes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>modifier programme</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Desktop/New folder/favicon.ico</normaloff>../../Desktop/New folder/favicon.ico</iconset>
  </property>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>19</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ID_Planning:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero_Sem:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>119</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Heure:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="Heure2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>118</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>69</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Date_Planing:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifierProgrammes_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>258</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Numero_Sem2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>209</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>NB_reservation:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>258</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom_Coach:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="Date_Planning2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>68</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Nom_Coach2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>259</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID_Planning2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>19</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="NB_reservation2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
